src/context/LogContext/index.tsx

```diff
@@ -180,9 +180,6 @@ const LogContextProvider: React.FC<LogContextProviderProps> = ({
       type: "SET_MATCH_COUNT",
       matchCount: results.length,
     });
```

Comment:
This is unrelated and I think has existed for a while, but uploading a file unexpectedly adds a bookmark to the n+1 line of the file.

Reply:
There is a ticket for this. https://jira.mongodb.org/browse/EVG-18350